README.md

```diff
@@ -247,13 +473,13 @@ This way, we don't need to copy ssh keys into the VM. You can verify that the ke
 running `ssh-add -l`.
 
 ```
-$ ssh-add -l
+zcash-gitian$ ssh-add -l
 The agent has no identities.
 
-$ ssh-add ~/.ssh/github_id_rsa
+zcash-gitian$ ssh-add ~/.ssh/github_id_rsa
 Identity added: /home/hpotter/.ssh/github_id_rsa (/home/hpotter/.ssh/github_id_rsa)
 
-$ ssh-add -l
+zcash-gitian$ ssh-add -l
 4096 SHA256:4fFdwJ71VIpF5cW0dqrsU7jxjctaFcAKmdQZPEqR0Y4 /home/hpotter/.ssh/github_id_rsa (RSA)
```

Comment:
Minor but the command to generate the ssh key would produce a 2048 bit key I think by default. Should it be updated to generate a 4096 bit key as per this one `ssh-keygen -t rsa -b 4096 -C "hpotter@hogwarts.wiz" -f ~/.ssh/github_id_rsa -N ''`

Reply:
seems reasonable; assuming this is meant for the line with the `ssh-keygen` command